buildbot/osuosl/master/config/builders.py

```diff
@@ -1754,24 +1754,26 @@
     'workernames' : ["minipc-1050ti-linux"],
     'builddir': "openmp-offload-cuda-project",
     'factory' : OpenMPBuilder.getOpenMPCMakeBuildFactory(
-                        clean=False,
-                        enable_runtimes=[],
+                        clean=True,
+                        enable_runtimes=['offload'],
```

Comment:
Is it not enabling `openmp` here on purpose?
(since the other builder _does_ explicitly enable it)

Reply:
`-DLLVM_ENABLE_PROJECTS=openmp` is the defining difference to the `openmp-offload-cuda-runtime` build. Otherwise, they would be identifical.